experimental/Schemes/MorphismFromRationalFunctions.jl

```diff
@@ -122,7 +122,9 @@ function Base.show(io::IOContext, Phi::MorphismFromRationalFunctions)
   if get(io, :supercompact, false)
     print("Morphism from rational functions")
   else
-    print("Hom: ", X, " -> ", Y)
```

Comment:
Fixes a bug, `X` and `Y` are not defined in this function

Reply:
(This also means there is no test for this printing variant, so ideally we should add one)